format/private/formatter_binary.bzl

```diff
@@ -14,7 +14,7 @@ TOOLS = {
     "Jsonnet": "jsonnetfmt",
     "Terraform": "terraform-fmt",
     "Kotlin": "ktfmt",
-    "Java": "java-format",
+    "Java": "prettier",
```

Comment:
Can we keep it as "java-format" and instead apply a `patches = []` attr to the MODULE.bazel file in cc-unified-storage?